weave-js/src/components/Sidebar/Outline.tsx

```diff
@@ -134,6 +127,33 @@ export const shouldDisablePanelDelete = (
   path.length === 0 ||
   (path.length === 1 && ['main', 'sidebar'].includes(path[0]));
 
+const PANEL_TYPE_TO_ICON: Record<string, IconName> = {
+  Group: 'group',
+  boolean: 'boolean',
+  number: 'number',
+  string: 'text-language-alt',
+  date: 'date',
+  DateRange: 'date',
+  FilterEditor: 'filter-alt',
+  table: 'table',
+  plot: 'chart-horizontal-bars',
+  Histogram: 'chart-vertical-bars',
+  object: 'list-bullets',
+  Expression: 'code-alt',
+};
+const getPanelTypeIcon = (panelId: string | undefined) => {
+  if (!panelId) {
+    return 'panel';
+  }
+  if (panelId.startsWith('row.')) {
```

Comment:
idk if theres ever a case with maybe.row something but i guess it would just default to panel which is fine

Reply:
I don't understand the row. panels as much as I'd like, but this sort of thing should be pretty easy to tweak.